packit_service/service/events.py

```diff
@@ -697,18 +697,33 @@ def __init__(
     ):
         super().__init__(project_url=project_url)
         self.action = action
+        # issue_id = ID of the specific comment
         self.issue_id = issue_id
+        # issue ID as is used while processing
         self.issue_iid = issue_iid
         self.repo_namespace = repo_namespace
         self.repo_name = repo_name
         self.project_url = project_url
         self.user_login = username
         self.comment = comment
-        self.commit_sha = None
+        self._tag_name = None
+
+    @property
+    def tag_name(self):
+        if not self._tag_name:
+            releases = self.project.get_releases()
+            self._tag_name = releases[0].tag_name if releases else ""
+        return self._tag_name
+
+    @property
+    def commit_sha(self):
+        return self.tag_name
```

Comment:
these two are indeed pretty cryptic, if you refactor them, comments/docstrings would be greatly appreciated!

Reply:
when doing documentation for `ogr` I noticed there's `get_latest_release` method, so I'll have a look into it and check how it behaves with no releases

Reply:
filed bug report, will fix tomorrow